Source/Fuse.iOS/iOSInterop.uno

```diff
@@ -56,5 +56,22 @@ namespace Fuse.iOS.Bindings
 				[[UIApplication sharedApplication] openURL:[NSURL URLWithString:uri]];
 			});
 		@}
+
+		[Foreign(Language.ObjC)]
+		public static extern(iOS) void LaunchApp(string uri, string appStoreUri)
+		@{
+			if ([[UIApplication sharedApplication] canOpenURL:[NSURL URLWithString:uri]]) 
+			{
+				dispatch_sync(dispatch_get_main_queue(), ^{
+					[[UIApplication sharedApplication] openURL:[NSURL URLWithString:uri]];
+				});
+			} 
+			else 
+			{
+				dispatch_sync(dispatch_get_main_queue(), ^{
+					[[UIApplication sharedApplication] openURL:[NSURL URLWithString:appStoreUri]];
+				});
+			}
```

Comment:
Do we need to check that `else if (appStoreUri != null && appStoreUri.length() > 0)` here? (This is pseudo-code and probably not valid Obj-C.)

Reply:
sorted.